src/context/MovieModalContext.tsx

```diff
@@ -9,12 +9,20 @@ interface MovieModalContextType {
 
 export const MovieModalContext = React.createContext<MovieModalContextType>({
   isOpen: false,
-  setIsOpen: () => {},
-  handleOpen: () => {},
+  setIsOpen: () => {
```

Comment:
You can extract this to a function let's say called

```
export const noop = () => () => {
  // do nothing
}
```

And then just import it and use it like

```
setIsOpen: noop()
```

Reply:
What the file name should be?

Reply:
`noop.ts` and it can be created inside your utils folder if you have one.